src/pages/index.js

```diff
@@ -34,69 +38,151 @@ const modalImageElement = imagePreviewModal.querySelector(".modal__image-preview
 const modalCaption = imagePreviewModal.querySelector(".modal__image-caption");
 const imagePreviewCloseButton = imagePreviewModal.querySelector(".modal__close");
 
+// Confirm Delete Modal
+const trashIcon = document.querySelector("#confirm-delete-modal");
+const trashSubmitButton = trashIcon.querySelector("#confirm-delete-submit")
 
+// Avatar Edit Modal
+const avatarEditModal = document.querySelector("#avatar-edit-modal");
 
 const cardTemplate = document.querySelector("#card-template").content.firstElementChild;
 
 
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "f34fc940-6444-4c84-9643-915e876889fb",
+    "Content-Type": "application/json",
+  }
+}); 
+
+let cardSection;
+
+api
+  .getInitialCards()
+  .then((res) => {
+    cardSection = new Section({
+      items: res,
+      renderer: (cardData) => {
+        const cardEl = renderCard(cardData);
+        cardSection.addItem(cardEl);
+      },
+    },  
+  ".cards__list"
+  );
+  
+  cardSection.renderItems();
+  })
+  .catch((err) => {
+    console.error(err); 
+  });
+  
+const profileUserInfo = new UserInfo(".profile__title", ".profile__description", ".profile__image");    
+  
+api
+  .loadUserInfo()
+  .then((res) => {
+    profileUserInfo.getUserInfo();
+  })
+  .catch((err) => {
+    console.error(err);
+  });
 
-// Create instances of the classes
-const cardSection = new Section({
-  items: initialCards,
-  renderer: (cardData) => {
-    const cardEl = renderCard(cardData);
-    cardSection.addItem(cardEl);
-  },
-},  
-".cards__list"
-);
 
-cardSection.renderItems();
 
-const profileEditFormPopup = new PopupWithForm("#profile-edit-modal", handleProfileEditSubmit);
-profileEditFormPopup.setEventListeners();
 
-const editFormElement = profileEditModal.querySelector(".modal__form_edit");
-const editFormValidator = new FormValidator(validationSettings, editFormElement);
-editFormValidator.enableValidation();
+const editFormElement = profileEditModal.querySelector(".modal__form_edit");  
+const editFormValidator = new FormValidator(validationSettings, editFormElement);  
+editFormValidator.enableValidation();  
 
-const profileAddCardPopup = new PopupWithForm("#profile-add-card-modal", handleProfileAddCardSubmit);
-profileAddCardPopup.setEventListeners();
+const avatarFormElement = avatarEditModal.querySelector(".modal__form_avatar");  
+const avatarFormValidator = new FormValidator(validationSettings, avatarFormElement);  

```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store the validator using the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```